src/ppx/scene/scene_material.cpp

```diff
@@ -126,6 +149,17 @@ scene::VertexAttributeFlags StandardMaterial::GetRequiredVertexAttributes() cons
     return attrFlags;
 }
 
+bool StandardMaterial::HasTextures() const
+{
+    bool hasBaseColorTex         = HasBaseColorTexture();
```

Comment:
that would be nice if c++ had the same 'default-is-immutable' as Rust 😥 I want to put const in front of each one of those variables, but it's noisy.

Reply:
Yeah, I agree. I've been adding const onto things slowly.